wiki/Tournaments/CWC/2022/en.md

```diff
@@ -99,6 +99,12 @@ The osu!catch World Cup 2022 is run by the [osu! team](/wiki/People/The_Team) an
 
 ## Match schedule: Grand Finals
 
+### Saturday, 2 July 2022
+
+| Team A | Team B | Match time |  |
+| --: | :-- | :-- | :-: |
+| All-stars Red | All-stars Blue | [Jul 3 (Sun) 12:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20220703T120000&p1=1440) | [^showmatch] |
```

Comment:
seems like wrong date cuz overlaps with grand finals, and it doesn't match your header

Reply:
I forgot to change the time for that table :/

Reply:
time still doesnt match o_O

Reply:
Fixed for good now